data/ignition/systemd/units/agent.service

```diff
@@ -0,0 +1,25 @@
+[Service]
+Type=simple
+Restart=always
+RestartSec=3
+StartLimitInterval=0
+Environment=HTTP_PROXY=
+Environment=http_proxy=
+Environment=HTTPS_PROXY=
+Environment=https_proxy=
+Environment=NO_PROXY=
+Environment=no_proxy=
+# TODO: If AUTH_TYPE != none, then PULL_SECRET_TOKEN needs to be updated
+# https://github.com/openshift/assisted-service/blob/master/internal/ignition/ignition.go#L1381
+Environment=PULL_SECRET_TOKEN={{.PullSecretToken}}
+TimeoutStartSec=180
+ExecStartPre=/usr/local/bin/agent-fix-bz1964591 quay.io/edge-infrastructure/assisted-installer-agent:latest
```

Comment:
Where does /usr/local/bin/agent-fix-bz1964591 get installed? Should this line be below the next one?

Reply:
Did some digging looks like this is added into the ignition files by [assisted-service](https://github.com/openshift/assisted-service/pull/1836).

[Bugzilla 1964591](https://bugzilla.redhat.com/show_bug.cgi?id=1964591)

Reply:
This line should have been removed. The script isn't needed because the appropriate fix has been made upstream in podman.